server/routes/viewReport.ts

```diff
@@ -66,3 +65,24 @@ export default function viewReport(service: Services): Router {
 
   return router
 }
+
+/**
+ * Replaces the questions/responses with the labels in the report config
+ */
+function useReportConfigLabels(report: ReportWithDetails, reportConfig: IncidentTypeConfiguration): ReportWithDetails {
+  for (const question of report.questions) {
+    const questionConfig = reportConfig.questions[question.code]
+    if (!questionConfig) {
+      // eslint-disable-next-line no-continue
+      continue
+    }
+
+    question.question = questionConfig?.label ?? question.code
```

Comment:
shouldn't the fallback be `?? question.question` at the end? that's the human readable bit as taken from nomis

Reply:
You are correct: The intent was that if for whatever reason the question/answer config couldn't be found (possibly mainly test data with bogus questions/responses) we would leave the question/response as returned from the API.

Reply:
Corrected here: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/287/commits/0f991237edffd7e8d58318ff3cbcab8e4cf5efb5